app/code/Magento/Catalog/Test/Mftf/Test/AdminCreateInactiveFlatCategoryAndUpdateAsInactiveTest.xml

```diff
@@ -37,9 +37,8 @@
             <magentoCLI stepKey="setFlatCatalogCategory" command="config:set catalog/frontend/flat_catalog_category 1"/>
             <!--Open Index Management Page and Select Index mode "Update by Schedule" -->
             <magentoCLI stepKey="setIndexerMode" command="indexer:set-mode" arguments="schedule" />
-            <!-- Run cron twice -->
-            <magentoCLI command="cron:run" arguments="--group=index" stepKey="runCron1"/>
-            <magentoCLI command="cron:run" arguments="--group=index" stepKey="runCron2"/>
+            <!-- Run cron -->
+            <magentoCLI command="cron:run" arguments="--group=index" stepKey="runCron"/>
```

Comment:
Can't we use https://devdocs.magento.com/mftf/docs/test/actions.html#magentocron instead?

Reply:
@ihor-sviziev  we can use it 
Thank you for advice.